src/js/views/panel_view.js

```diff
@@ -219,11 +219,12 @@
                             format = prop_nf[1] ? prop_nf[1] : "index";
                             precision = param_dict["precision"] !== undefined ? param_dict["precision"] : 0; // decimal places default to 0
                             label_value = self.model.get_time_label_text(format, param_dict["offset"], precision);
-                        } else if (['image', 'dataset'].includes(prop_nf[0])){
+                        } else if (['image', 'dataset', 'screen', 'plate', 'well', 'wellsample'].includes(prop_nf[0])){
```

Comment:
Could you add `acquisition` to the list of accepted objects? It's already part of the parent in the JSON response.
 
And/or the alias `run` with a ternary condition inside the if:
`prop_nf[0] = prop_nf[0] == "run" ? "acquisition" : prop_nf[0]`